spec.html

```diff
@@ -19731,9 +19731,9 @@ <h1>Runtime Semantics: ClassDefinitionEvaluation</h1>
           1. Let _constructorParent_ be the intrinsic object %FunctionPrototype%.
         1. Else,
           1. Set the running execution context's LexicalEnvironment to _classScope_.
-          1. Let _superclass_ be the result of evaluating |ClassHeritage|.
+          1. Let _superclassRef_ be the result of evaluating |ClassHeritage|.
           1. Set the running execution context's LexicalEnvironment to _lex_.
-          1. ReturnIfAbrupt(_superclass_).
+          1. Let _superclass_ be ? GetValue(_superclassRef_).
```

Comment:
Couldn't _superclassRef_ also be an abrupt completion here?

Reply:
Yes, but [GetValue](https://tc39.github.io/ecma262/#sec-getvalue) handles that gracefully by having `ReturnIfAbrupt` as its first step.